### PR TITLE
Bugfix: database error as string not handled

### PR DESCRIPTION
**Pull request by r-peschke**

### CODE REVIEW

File: openslides_backend/services/datastore/handle_datastore_errors.py [hunk 1]
```diff
@@ -34,31 +34,39 @@ def raise_datastore_error(
...
+                raise DatastoreLockedException(error_message)
+            elif type_verbose == "MODEL_DOES_NOT_EXIST":
+                error_message = " ".join(
+                    (
+                        error_message,
+                        f"Model '{additional_error_message.get('fqid')}' does not exist.",
+                    )
+                )
+            else:
+                error_message = " ".join((error_message, str(additional_error_message)))
```
Comment (reiterl) [inline openslides_backend/services/datastore/handle_datastore_errors.py:69]: Doesn't look right, why is this same error_message twice?

File: openslides_backend/services/datastore/handle_datastore_errors.py [hunk 2]
```diff
@@ -34,31 +34,39 @@ def raise_datastore_error(
...
-                (
-                    error_message,
-                    f"The following locks were broken: {broken_locks}",
+        if isinstance(additional_error_message, dict):
+            type_verbose = additional_error_message.get("type_verbose", "")
+            if type_verbose == "MODEL_LOCKED":
+                broken_locks = (
+                    "'"
+                    + "', '".join(
+                        sorted(cast(str, additional_error_message.get("keys")))
```
Comment (reiterl) [inline openslides_backend/services/datastore/handle_datastore_errors.py:46]: It doesn't look right here.

**Review by reiterl — COMMENTED** [review]
Here are no tests. They are missing.

File: openslides_backend/services/datastore/handle_datastore_errors.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/services/datastore/handle_datastore_errors.py:46]: This cast to str doesn't make sense.

File: openslides_backend/services/datastore/handle_datastore_errors.py (hunk 1 above)
Comment (r-peschke) [reply openslides_backend/services/datastore/handle_datastore_errors.py:69]: IMO it is easier this way. 
The first one is to append the additional_error_message, which is a dict, but only conditionally.
The second one is to append it, if it is not a dict, but without any conditions. 
It is by chance that the statement is identical, the str-function takes all types as argument.

Comment (reiterl) [reply openslides_backend/services/datastore/handle_datastore_errors.py:69]: Okay, accepted.

File: openslides_backend/services/datastore/handle_datastore_errors.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/services/datastore/handle_datastore_errors.py:46]: Okay, we discussed it and use the Iterable[str].

**Pull request merged**